my-website/docs/competicoes/sacup/designtestevaluation.md

```diff
@@ -6,6 +6,673 @@ sidebar_label: Design, Test & Evaluation Guide
+
+Os sistemas de propulsão híbridos e líquidos devem
+implementar um meio para ventilação ou descarga controlada remotamente de todos os líquidos e gases
+propelentes em caso de abortamento do lançamento.
+
+### Eletrônica do circuito de ignição por estágio / partida a ar 
+
+Todos os sistemas de ignição de estágio superior devem estar em conformidade com
+os mesmos requisitos e objetivos para “fiação crítica de segurança” dos sistemas de recuperação.
+
+● Os voos encenados devem ter uma relação empuxo-peso mínima de 8 no boost.
+
+● O sustentador deve ter uma relação empuxo-peso mínima de 3.
+
+● Os foguetes agrupados devem ter uma relação empuxo-peso mínima de 6 em qualquer motor que seja
+acendeu na plataforma (ou seja, o foguete deve ser capaz de voar com segurança se apenas um dos vários
+luzes de motores).
+
+### Requisitos do computador para a ignição por estágio / partida a ar 
+
+A ignição dos motores air-start deve ser
+realizado usando um computador de vôo COTS que tem a capacidade de realizar um "check de altitude" que pode inibir a ignição da partida abaixo de uma altitude pré-selecionada.
+
+Os atuais aprovados estão no documento oficial.
+
+### Voos Air-Start / Staged
+
+
+Todos os projetos devem ter disposições capazes de
+impedir a ignição do motor de partida a ar no solo.
+
+● Uma provisão para abrir o circuito entre o computador de vôo e o iniciador durante
+a inicialização do computador de vôo é obrigatória.
+
+● Shunts são recomendados, mas não são obrigatórios. Exemplos de troca recomendada
+projetos são fornecidos no Apêndice D.
+
+● Os computadores de vôo não devem ser armados até que o foguete esteja na posição vertical.
+
+● A configuração eletrônica deve ser projetada de tal forma que a provisão usada para abrir o
+circuito para o iniciador pode ser usado para abrir novamente o circuito para o iniciador no evento
+que o foguete não foi lançado.
+
+### Inibição Durante o vôo 
+
+O computador de vôo que controla o air-start
+a ignição do motor deve ser configurado para inibir a ignição do motor de partida a ar, a menos que o reforço
+burnout foi detectado e o foguete atingiu uma altitude de pelo menos 80% do
+altitude simulada no momento em que o disparo do iniciador é desejado.
+
+● O computador de vôo deve ser configurado para evitar que o motor de partida a ar atire em um
+mais tarde, se o limite de altitude não foi alcançado.
+
+### Teste do Sistema de Propulsão SRAD
+
+As equipes devem cumprir todas as regras, regulamentos e melhores
+práticas impostas pelas autoridades em seus locais de teste escolhidos.
+
+● Os requisitos a seguir referem-se ao teste de verificação do aluno pesquisado e
+desenvolveu (SRAD) e modificou os sistemas de propulsão disponíveis no mercado (COTS).
+
+● A ESRA recomenda que as equipes concluam esses testes até 1º de abril do ano de competição.
+
+### Teste de pressão da câmara de combustão
+SRAD e sistema de propulsão COTS modificado
```

Comment:
Essa tradução também ficou esquisita. Se for usar o google tradutor, dá uma revisada depois e 
a) Coloca termos mais comuns em inglês de volta e põe a tradução dentro de parênteses
ex: Student Research and Developed(SRAD, ou solução Pesquisada e Desenvolvida pelos Estudantes)
b) Corrige traduções mal feitas